pkg/model/crd.go

```diff
@@ -657,6 +657,45 @@ func (r *CRD) GetAllRenames(op OpType) (map[string]string, error) {
 	return renames, nil
 }
 
+// FindIdentifierInShape returns true and the resource's identifier iff shape
+// contains a matching field or corresponding plural meaning it can be populated
+// using the resource identifier.
+func (r *CRD) FindIdentifierInShape(
+	shape *awssdkmodel.Shape) (string, bool) {
+	resIdentifier := ""
+	found := false
+	resourceIdentifiers := []string{
```

Comment:
Should these have capitalised initialisms? i.e. `r.Names.Original + "ID"`

I think our spec and status field keys will have proper casing.

Reply:
I don't think capitalized initialisms are needed. Using the existing `CheckRequiredFieldsMissingFromShape` as reference, when the Spec/Status field path is being constructed the lookup in Spec/Status is done using memberName (VpcId) not cleanMemberName (i.e. VPCID). I'll dive deeper to verify though